bin/one-time-setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Runs once after devcontainer is created
+
+figlet "1-time Setup"
+echo "Running one-time setup for you :)"
+
+# Create .env files if it doesn't exist
+if [ ! -f ./client/.env ]; then
+    cp ./client/.env.example ./client/.env
+fi
+if [ ! -f ./server/.env ]; then
+    cp ./server/.env.example ./server/.env
+fi
+
+# if node_modules doesn't exist, install it
+if [ ! -d ./client/node_modules ]; then
+    (cd client && npm install)
+fi
+# TODO Figure out why this env variable is not getting loaded
+# probably because it needs to be added in .zprofile
+export POETRY_VIRTUALENVS_CREATE=false
```

Comment:
The postCreateCommand and postStartCommand doesn't pass down env vars. I couldn't figure it out either

Reply:
Or maybe because the shebang at the top is bash, so it doesn't use zshrc 😅😅... It has to use bashrc 😅😅😅